webrtc.html

```diff
@@ -2244,13 +2226,37 @@ <h4>
                                   </li>
                                   <li>
                                     <p>
-                                      For each of the codecs that <var>description</var> negotiates for sending, execute the following steps:
+                                      Clear the <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[NegotiatedCodecs]]}} slot.
+                                      <br>
+                                      Then, for each of the codecs that <var>description</var> negotiates for sending, execute the following steps:
                                       <ol>
                                         <li>Locate the matching codec description in <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendCodecs]]}}.</li>
                                         <li>If the matching codec description is not found, abort these steps.</li>
```

Comment:
Which steps are aborted here? One of the nested for-loops or sLD/sRD without error? Either seems wrong.

Not failing assumes negotiated codecs is a subset of this client's [[SendCodecs]]. The [1.0 spec](https://github.com/w3c/webrtc-pc/issues/2956#issuecomment-2072500942) didn't have this:

<img width="627" alt="image" src="https://github.com/user-attachments/assets/3e8d1c84-179c-4ef5-8107-d6bd6a249a29">

...which, AFAIK meant `sender.getParameters().codecs` might include _remote_ codecs not found locally, but found in the remote answer, which [JSEP](https://rtcweb-wg.github.io/jsep/#rfc.section.5.3.1) seems to allow: _"Any currently available media formats that are not present in the current remote description MUST be added after all existing formats."_

Why is this new limitation needed?

It's also unclear what constitutes a "matching codec description".